librdmacm/man/udpong.1

```diff
@@ -0,0 +1,62 @@
+.\" Licensed under the OpenIB.org BSD license (FreeBSD Variant) - See COPYING.md
+.TH "UDPONG" 1 "2017-04-28" "librdmacm" "librdmacm" librdmacm
+.SH NAME
+udpong \- unreliable datagram streaming over RDMA ping-pong test.
+.SH SYNOPSIS
+.sp
+.nf
+\fIudpong\fR [-s server_address] [-b bind_address]
+			[-B buffer_size] [-C transfer_count]
+			[-S transfer_size] [-p server_port] [-T test_option]
+.fi
+.SH "DESCRIPTION"
+Uses unreliable datagram streaming over RDMA protocol (rsocket) to
+connect and exchange data between a client and server application.
+.SH "OPTIONS"
+.TP
+\-s server_address
+The network name or IP address of the server system listening for
+connections.  The used name or address must route over an RDMA device.
+This option must be specified by the client.
+.TP
+\-b bind_address
+The local network address to bind to.
+.TP
+\-B buffer_size
+Indicates the size of the send and receive network buffers.
+.TP
+\-C transfer_count
+The number of messages to transfer from the client to the server and
+back again on each iteration.  (default 1000)
+.TP
+\-S transfer_size
+The size of each send transfer, in bytes.  (default 1000)
+.TP
+\-p server_port
+The server's port number.
+.TP
+\-T test_option
+Specifies test parameters.  Available options are:
+.P
+s | socket  - uses standard socket calls to transfer data
+.P
+a | async - uses asynchronous operation (e.g. select / poll)
+.P
+b | blocking - uses blocking calls
+.P
+n | nonblocking - uses non-blocking calls
+.P
+e | echo - server echoes all messages
+.SH "NOTES"
+Basic usage is to start udpong on a server system, then run
+udpong -s server_name on a client system.  udpong
+will run a series of latency and bandwidth performance tests.
```

Comment:
wait - does this do streaming datagram transfers too?  LOL - I can't remember what I wrote for this test anymore.